README.md

```diff
@@ -15,18 +15,17 @@ Follow these steps to add a new version:
 
 Generate API docs in Foreman:
 
-1. cd to foreman directory and checkout the relevant stable branch
-1. disable any plugins installed locally
-1. `APIPIE_RECORD=examples bundle exec rake test`
-1. `FOREMAN_APIPIE_LANGS=en bundle exec rake apipie:cache`
+1. Go to https://github.com/theforeman/foreman/actions/workflows/foreman.yml?query=branch%3AX.Y-stable
+1. Pick the latest run of the action
+1. Download the `apidoc-*` artifact (there might be multiple, pick any one of them)
```

Comment:
Does it make sense to download them automatically using `gh`?
```
gh run download --repo theforeman/foreman --name 'apidoc-*' $(gh run list --repo theforeman/foreman --workflow foreman.yml --branch 3.16-stable --status completed --limit 1 --json databaseId --jq .[].databaseId)
```

Reply:
probably a good example how to do this (but I'd not force the usage of `gh` I guess?)

Reply:
btw, it needs to be `--pattern`, not `--name`